databox/indexer/src/handlers/phraseanet/indexer.ts

```diff
@@ -137,31 +230,78 @@ export const phraseanetIndexer: IndexIterator<PhraseanetConfig> =
             }
 
             const searchParams = {
-                bases: databoxCollections[dm.databoxId],
+                bases: sourceCollections,
             };
+
+
+            // create stories as collections
+            //
+            logger.info(`Fetching stories`);
+            const recordStories: Record<string, string[]> = {};   // key: record_id ; values: story_id's
+            if(storiesCollectionId !== null) {
+                let stories: PhraseanetStory[] = [];
+                let offset = 0;
+                do {
+                    stories = await client.searchStories(searchParams, offset, "");
+                    for (const s of stories) {
+                        const storyCollId = await databoxClient.createCollection(
+                            s.resource_id,
+                            {
+                                workspaceId: workspaceId,
+                                key: idempotencePrefixes["collection"] + s.databox_id + "_" + s.story_id,
+                                title: s.title,
+                                parent: "/collections/" + storiesCollectionId
+                            }
+                        );
+                        logger.info(
+                            `Phraseanet story "${s.title}" (#${
+                                s.story_id
+                            }) from base "${databox.collections[s.base_id].name}" (#${
+                                s.base_id
+                            }) ==> collection (#${storyCollId})`
+                        );
+                        for (const r of s.children) {
+                            if (recordStories[r.record_id] === undefined) {
+                                recordStories[r.record_id] = [];
+                            }
+                            recordStories[r.record_id].push(storyCollId);
```

Comment:
we only have record_id here -not a full record-, so to create the asset here we need a request to fetch it.
since the same record can be in many stories that's many api calls (or a cache in ram... same risk of memory overflow).
todo: store this (and more) in a db (will also help stop / restart).